dali/python/backend_impl.cc

```diff
@@ -223,7 +224,8 @@ void ExposeTensorList(py::module &m) { // NOLINT
   // the backend. We do not support converting from TensorLists
   // to numpy arrays currently.
   py::class_<TensorList<CPUBackend>>(m, "TensorListCPU", py::buffer_protocol())
-    .def("__init__", [](TensorList<CPUBackend> &t, py::buffer b) {
+    .def("__init__", [](TensorList<CPUBackend> &t, py::buffer b,
+                        DALITensorLayout layout) {
```

Comment:
I think we should stay have some most used default (the one that DALI uses internally = NHWC)

Reply:
to be considered in a follow up